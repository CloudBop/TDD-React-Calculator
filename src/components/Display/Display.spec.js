import React from 'react';
import { shallow } from 'enzyme';
import Display from './Display'

describe('Calculator', () => {
  let wrapper;

  beforeEach(() => wrapper = shallow(<Display displayValue={""} />));
  // it('Should compare against snapshot <Calculator/>', () => expect(wrapper).toMatchSnapshot())
  
  it('should render a <div />', () => {
    expect(wrapper.find('div').length).toEqual(1);
  });

  
});