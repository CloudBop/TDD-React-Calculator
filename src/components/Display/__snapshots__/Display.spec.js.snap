// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Unit Tests Should compare against snapshot <Display/> 1`] = `ShallowWrapper {}`;
